### ISSUE TITLE
sam delete: allow to pass s3_bucket and s3_prefix as arguments

### ISSUE THREAD

**Issue by pjungermann** [aws/aws-sam-cli#3246]
### Describe your idea/feature/enhancement

At `sam delete` introduced at SAM CLI v1.29.0, you can delete your stack including resources like the ECR image as well as S3 objects.

Currently, you cannot pass the S3 bucket and prefix to the `sam delete` command. It is only obtained from `samconfig.toml` using the "deploy" scope. See the "workaround" at the "Additional Details" section.

If there is no such config, the deletion of S3 objects will be skipped.

Example output
```
2021-09-06 08:23:31,299 | Cannot delete s3 files as no s3_bucket found

Warning: s3_bucket and s3_prefix information could not be obtained from local config file or cloudformation template, delete the s3 files manually if required
```

### Proposal

Add arguments `--s3-bucket` and `--s3-prefix` to the command `sam delete` as they exist e.g. at `sam deploy`.

### Additional Details

Workaround:
```
cat << TOML > samconfig.toml
version=0.1
[default.deploy.parameters]
stack_name = "${STACK_NAME}"
s3_bucket  = "${S3_BUCKET}"
s3_prefix  = "${S3_PREFIX}"
TOML
```

If stack_name is not part of the config or it is not matching the stack name you may have passed as `--stack-name` argument, the config will **not** be used.

**Comment by aahung** [comment]
@pjungermann thanks for the suggestion

**Comment by qingchm** [comment]
I believe that this is addressed in the new 1.53.0 release. Please have a try and download our latest version of SAMCLI. If you still have any questions or issues feel free to re-open the issue!
